cypress/integration/lpa/cases/tasks.spec.js

```diff
@@ -0,0 +1,100 @@
+describe('Create a task', { tags: ["@lpa", "@smoke-journey"] }, () => {
```

Comment:
This 'Create a task' test already exists, but appears as a change I made because I renamed the file